### DISCUSSION TITLE
Support for supabase

### DISCUSSION THREAD

**Discussion by bigOK666** [nextify-limited/saasfly#7] [Q&A]
Hi, thanks for sharing this amazing template. Is supporting supabase in the plan?

**Comment by nextify2025** [comment]
Can you tell me about a scenario where Supabase can be used ？ User Admin & Auth  or more simple database operations ? I tried to integrate him in the past, but the over-abstraction and complexity and the maintainability it brought didn't convince me to choose him 🙏

**Comment by bigOK666** [comment]
for me it is Admin & Auth & user related data at a very good price. More details [here](https://www.reddit.com/r/nextjs/comments/13oksux/vercel_postgres_vs_supabase/)

btw, I love your profile pic because I am a huge fan of GINTAMA

**Comment by nextify2025** [comment]
Thank you very much for your feedback! Regarding the three requirements you raised, we have the following preliminary plans:

1. **Admin Dashboard**: We plan to add a simple admin dashboard to view operational data. This will help us better understand product usage and provide a basis for subsequent optimization.

2. **User Authentication**: We tend to integrate lucia-auth as the user authentication solution. Lucia is a lightweight, scalable authentication library that integrates well with the Next.js ecosystem. However, we will also evaluate the authentication services provided by Supabase and choose the best one.

3. **User-Related Data**: The database service provided by Supabase seems a great choice. We'll consider priority integration

Regarding the specific development timeline, we are currently unable to give an exact commitment. Product development is not a one-step process; it is a gradual process. We tend to collect user feedback first, and on this basis, gradually refine the requirements before entering the development phase.
